mapo/agents/mapo/tests/test_model.py

```diff
@@ -0,0 +1,194 @@
+# pylint: disable=missing-docstring, redefined-outer-name
+import pytest
+from gym.spaces import Box
+import numpy as np
+import tensorflow as tf
+
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.models.catalog import ModelCatalog
+
+from mapo.agents.mapo.mapo_model import MAPOModel
+
+
+def get_spaces():
+    return (
+        Box(-np.inf, np.inf, shape=(2,), dtype=np.float32),
+        Box(-1.0, 1.0, shape=(2,), dtype=np.float32),
+    )
+
+
+def get_models():
+    obs_space, action_space = get_spaces()
+    model_config = {
+        "custom_options": {
+            "actor": {"activation": "relu", "layers": [32, 32]},
+            "critic": {"activation": "relu", "layers": [32, 32]},
+            "dynamics": {"activation": "relu", "layers": [32, 32]},
+        }
+    }
+    models = [
```

Comment:
Just a suggestion... you may find convenient to use `itertools.product([True, False], [True, False])` to generate all cross product combinations for the `target_networks` and `twin_q` aguments... something along the lines of:

```python3
models = [
    MAPOModel(
            obs_space,
            action_space,
            1,
            model_config,
            "test_model",
            target_networks=target_networks,
            twin_q=twin_q
    )
    for target_networks, twin_q in itertools.product([False, True], [False, True])
]
```

Reply:
Sounds good!